phpunit/functional/PendingReasonTest.php

```diff
@@ -724,7 +837,175 @@ public function testHandlePendingReasonUpdateFromNewTimelineItem(): void
                     $correct_timeline_item->getID(),
                     $last_timeline_item_pending_data->fields['items_id']
                 );
+                $this->assertEquals(
+                    $last_timeline_item_pending_data->fields['last_bump_date'],
+                    $ticket_pending_data->fields['last_bump_date']
+                );
+            }
+        }
+    }
+
+    protected function testCronPendingReasonAutobumpAutosolveProvider()
+    {
+        $this->login();
+        $entity = getItemByTypeName('Entity', '_test_root_entity', true);
+
+        $currentDate = date('Y-m-d H:i:s');
```

Comment:
It seems yours test fail with a 1 second difference right now.

You can use `Session::getCurrentTime()` in your test instead of calling `date` to get a stable value that will not change during the execution (make the tests more reliant).

If you want, you can also go a step further and set `$_SESSION['glpi_currenttime']` to set a given date that will not change at all so the test is even more predictable and you do not need to do relative computations.